CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Changelog
+### 0.9.4
+- **Add**: `SessionManager.request` method, that takes an array as a request parameter.
```

Comment:
add info about NetworkServiceConfiguration.apiRequestParameters and ApiRequestParameters.init